docusaurus.config.js

```diff
@@ -149,9 +149,9 @@ const config = {
       // Styles for this are controlled in src/css/announcement-bar.scss
       announcementBar: {
         //give id a unique value to get a new announcement bar to appear
-        id: 'cypress-v13',
+        id: 'cypress-v13+',
```

Comment:
intentional change?

Reply:
yep. but unrelated to the safari bit. I'll pull it from this PR